version.php

```diff
@@ -29,8 +29,8 @@
     die('Direct access to this script is forbidden.'); // It must be included from a Moodle page.
 }
 
-$plugin->version   = 2021011911;      // The current plugin version (Date: YYYYMMDDXX).
-$plugin->release   = 2021011910;      // Same as version
+$plugin->version   = 2021122110;      // The current plugin version (Date: YYYYMMDDXX).
+$plugin->release   = 2021122110;      // Same as version
```

Comment:
```suggestion
$plugin->release   = 2021122110;      // Same as version.
```
Would be good to remove the PHPCS warning on this line as well